### PR TITLE
Add phone and bundle strings

### PR DESCRIPTION
**Pull request by Vinnl**
For those, Spanish and French translation is enough.

Note that some additional work might need to be done for Pontoon to migrate properly before merging this PR: https://github.com/mozilla-l10n/fx-private-relay-l10n/issues/104.

### CODE REVIEW

**Comment by flodolo** [comment]
@peiying2 
Just to confirm, have you synced with @mathjazz around this? I would strongly suggest to test it on stage, if that's not already planned, before merging for prod.

File: en/phones.ftl
Comment (peiying2) [inline en/phones.ftl:91]: duplicate `your`

File: en/phones.ftl
Comment (peiying2) [inline en/phones.ftl:140]: Is there a reason this string is broken into two lines?

File: en/phones.ftl
Comment (peiying2) [inline en/phones.ftl:75]: add a period at the end of the string.

File: en/phones.ftl
Comment (peiying2) [inline en/phones.ftl:113]: Is there anything after `Can't find it?`

**Review by peiying2 — COMMENTED** [review]
Do NOT merge before Pontoon migration of translated string is completed. 

In addition to the individual brand names, the entries are repeated for every file. If we were to add entries in the future, all on-production files will be added. They need to be individual submitted even though most don't require translation. Let's consider creating a brand file if it is not too much engineering work.

**Comment by Vinnl** [comment]
> the entries are repeated for every file

This is just referring to the brand strings, right? Do you have a link to documentation on how to do that, or alternatively an example repo that has done this? Happy to implement that if I can find out how to :)

> If we were to add tries in the future, all on-production files will be added. They need individual submitted even though most don't require translation.

I'm sorry, I don't understand this sentence. What does "tries" mean here?

File: en/phones.ftl
Comment (Vinnl) [reply en/phones.ftl]: 2a33745

File: en/phones.ftl
Comment (Vinnl) [reply en/phones.ftl]: Woah, not even QA caught that: 2a33745

File: en/phones.ftl
Comment (Vinnl) [reply en/phones.ftl]: Don't think so: 2a33745

File: en/phones.ftl
Comment (Vinnl) [reply en/phones.ftl]: 2a33745

File: en/phones.ftl
Comment (Vinnl) [reply en/phones.ftl:114]: Yeah, but it's on a separate line it looks like:

image

I added a comment to clarify that in 2a33745, is that enough? I'm not sure if I have a good way of incorporating those into the same string.

**Comment by peiying2** [comment]
> I'm sorry, I don't understand this sentence. What does "tries" mean here?

Looks like you already did what I meant to suggest you to do. Sorry for the incoherent comment, lol - not sure what happened - can't blame the fat fingers. I meant to say the individual brand names (not comments), and "entries" instead of "tries". You already took care of it. Thank you!

**Comment by peiying2** [comment]
LGTM! Thanks for the quick update. May need a dev to check the section on the minute/second remaining to verify number. Want to make sure the logic is correct.

**Comment by Vinnl** [comment]
@peiying2 OK, and are you reaching out to that dev (is that mathjazz?)? Also, does flod's "testing on stage" mean a stage env for Pontoon, or just the regular Relay stage environment? We'll definitely do the latter, but I don't know how to do the former.

(I'm also adding a couple of commits with some more new strings.)

**Comment by peiying2** [comment]
> @peiying2 OK, and are you reaching out to that dev (is that mathjazz?)? Also, does flod's "testing on stage" mean a stage env for Pontoon, or just the regular Relay stage environment? We'll definitely do the latter, but I don't know how to do the former.
> 
> (I'm also adding a couple of commits with some more new strings.)

For testing of added files, the staging is on Pontoon to make sure the files show up only in the locales specified. 

For the code on the time (hr/sec), if your team has already reviewed it, and it looks correct, then I am good. On our side, it would be flod, but he is off this week.

**Comment by Vinnl** [comment]
On our side, those strings show up fine in English, so I think we're good. Does that mean that once https://github.com/mozilla-l10n/fx-private-relay-l10n/issues/104 is merged, there are no more blockers for this one to be merged too?

**Comment by mathjazz** [comment]
I have tested this patch locally to verify the migration plan outlined in #104 works as expected and it does. :)

I've noticed a few things though, which I think we should fix before we proceed:

1. Seems like the newly created `brands.ftl` file is not referenced in the TOML file, which means it will not be exposed to any locale. I guess that's not intentional?

1. For future reference, please note that moving source strings from one file to another (e.g. `brands.ftl`) also requires moving any existing translations corresponding to these strings. Otherwise we'll lose them. For this patch, I'll do the migration of translations in the DB.

1. As per convention, I'd also suggest we move the TOML file to the root of the repository and rename it to l10n.toml.

**Comment by Vinnl** [comment]
Whoops, thanks for the check @mathjazz, still have to get used to the TOML file :) Hopefully we shouldn't need to move strings from one file to another after this is merged.

I also renamed the TOML file as per your suggestion.

**Comment by Vinnl** [comment]
@peiying2 I updated a couple of strings for https://github.com/mozilla/fx-private-relay/pull/2649: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/103/commits/a8b16dec29bccde1596db9b5c8c827f1ae4905fd

**Comment by peiying2** [comment]
> @peiying2 I updated a couple of strings for [mozilla/fx-private-relay#2649](https://github.com/mozilla/fx-private-relay/pull/2649): [a8b16de](https://github.com/mozilla-l10n/fx-private-relay-l10n/commit/a8b16dec29bccde1596db9b5c8c827f1ae4905fd)

The added strings look good.

File: l10n.toml
Comment (mathjazz) [reply l10n.toml:30]: It's a good practice, but it's not required.

**Review by mathjazz — APPROVED** [review]
I tested the latest changes and it worked as expected locally.

I'll merge now and do the migration in Pontoon.

**Pull request merged**
